packages/cli/src/util/deploy/create-git-meta.ts

```diff
@@ -9,10 +9,11 @@ import { Output } from '../output';
 export function isDirty(directory: string, output: Output): Promise<boolean> {
   return new Promise(resolve => {
     exec('git status -s', { cwd: directory }, function (err, stdout, stderr) {
-      if (err) return resolve(false);
-      if (stderr) {
+      if (err || stderr) {
         output.debug(
-          `Failed to determine if git repo has been modified: ${stderr.trim()}`
+          `Failed to determine if git repo has been modified:\n${err}\n${
+            stderr && stderr.trim()
```

Comment:
if `err` is `null`, there will be two newlines here. Can we optionally add these values to the resulting string?